contrib/inventory/ec2.py

```diff
@@ -672,7 +672,9 @@ def get_elasticache_clusters_by_region(self, region):
                 error = self.get_auth_error_message()
             if not e.reason == "Forbidden":
                 error = "Looks like AWS ElastiCache is down:\n%s" % e.message
-            self.fail_with_error(error, 'getting ElastiCache clusters')
+                self.fail_with_error(error, 'getting ElastiCache clusters')
+            else:
+                return
```

Comment:
I don't know, but neither do I care, if I don't have the perms, whether there are any RDS instances.

The default ec2.ini should be suitable for most purposes, and not error in the legitimate case that a locked down user exists.